01_Exercises/LAB_03.py

```diff
@@ -0,0 +1,171 @@
+import os
+
+
+def do_list_integers(random_nr):
+    """
+    :param: must be a random number or a string number.
+    :return: a list of integers;
+    :note: The function will fail if the string is not numeric;
+    """
+    random_nr = str(random_nr)
+    random_list = list(random_nr)
+    all_letters = ['a', 'b', 'c', 'd', 'e', 'f', 'g', 'h', 'i', 'j', 'k', 'l', 'm', 'n', 'o', 'p', 'q', 'r', 's', 't',
+                   'u', 'v', 'w', 'x', 'y', 'z']
+
+    for num in random_list:
+        if num in all_letters:
+            return "This random number contains letters, please try again!"
+        else:
+            pass
+
+    for num in range(len(random_list)):
+        if random_list[num] == "-":
+            random_list[num + 1] = int(random_list[num + 1]) * -1
+
+    for num in random_list:
+        if num == "-":
+            random_list.remove(num)
+
+    random_list = list(map(int, random_list))
```

Comment:
De ce mai aplici inca odata `int` peste toate numerele daca ai aplicat deja ?

Reply:
Pai nu am aplicat la toate. Am aplicat doar la numerele negative.
Daca returnez random_list fara sa fac maparea aia (sa transform in integers din string).
Atunci random_list o sa contina si stringuri si integers.
Iar in descriere scrie ca functia returneaza doar integers. (de cand asteptam sa te torn :))  cu codul meu minune)